nixos/modules/services/cluster/kubernetes/pki.nix

```diff
@@ -148,6 +148,18 @@ in
         address = "0.0.0.0";
         tlsCert = cfsslCert;
         tlsKey = cfsslKey;
+        disable = [
```

Comment:
Practically, `certmgr` only uses the `info`, `authsign` and `sign` endpoints and that's what we leave enabled.

Ideally, we should be using `multirootca` instead of `cfssl serve` which does properly restrict the endpoints to only the authenticated ones. That's a bit more involved though and this is easier to put in a PR.